great_expectations/cli/cli.py

```diff
@@ -91,7 +91,7 @@ def get_command(self, ctx: click.Context, name: str) -> Optional[str]:  # type:
             return None
 
 
-@click.group(cls=CLI, name="great_expectations")
+@click.command(cls=CLI, name="great_expectations")
```

Comment:
This looks to be a real error in our code that is now being caught. Here are the docs for multicommands and groups: https://click.palletsprojects.com/en/8.1.x/commands/#custom-multi-commands

Reply:
that's with v8 - I think this may have been different before?

Reply:
I've run mypy using click 7.1.2 and this passes. Looking at [their release notes](https://click.palletsprojects.com/en/8.1.x/changes/), it looks like they have done some work with typing recently.